hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestSstFilteringService.java

```diff
@@ -309,4 +311,21 @@ private static OmKeyArgs createAndCommitKey(OzoneManagerProtocol writeClient,
     return keyArg;
   }
 
+  @Test
+  public void testFilterFunction() {
+    Assert.assertTrue(SstFilteringService.FILTER_FUNCTION.apply(
+        "/vol1/bucket1/key1",
+        "/vol1/bucket1/key1",
+        "/vol1/bucket1/"));
+
+    Assert.assertFalse(SstFilteringService.FILTER_FUNCTION.apply(
+        "/vol1/bucket1/key1",
+        "/vol1/bucket1/key1",
+        "/vol1/bucket2/"));
+
+    Assert.assertFalse(SstFilteringService.FILTER_FUNCTION.apply(
+        "/vol1/bucket1/key1",
+        "/vol1/bucket1/key1",
+        "/vol1/bucket/"));
+  }
```

Comment:
```suggestion
  }    Assert.assertTrue(SstFilteringService.FILTER_FUNCTION.apply(
        "/vol1/bucket1/key1",
        "/vol1/bucket1/key80",
        "/vol1/bucket1/"));

    Assert.assertTrue(SstFilteringService.FILTER_FUNCTION.apply(
        "/vol1/bucket1/key1",
        "/vol1/bucket5/key1",
        "/vol1/bucket3/"));

    Assert.assertFalse(SstFilteringService.FILTER_FUNCTION.apply(
        "/vol1/bucket1/key1",
        "/vol1/bucket4/key9",
        "/vol1/bucket5/"));
  }
```
Maybe add  cases where first & last keys are different

Reply:
Added tests as suggested. Thanks @sadanand48.